src/test/components/TrackContextMenu.test.js

```diff
@@ -825,6 +826,84 @@ describe('timeline/TrackContextMenu', function () {
     );
   });
 
+  describe('show all local tracks in a process', function () {
+    function setupMoreTracks() {
+      const profile = getProfileWithMoreNiceTracks();
+      const store = storeWithProfile(profile);
+
+      render(
+        <Provider store={store}>
+          <TimelineTrackContextMenu />
+        </Provider>
+      );
+
+      function clickAllThreadPoolTracks() {
+        const threadPoolTracks = screen.getAllByText(/^ThreadPool#\d$/);
+        for (const track of threadPoolTracks) {
+          fireFullClick(track);
+        }
+      }
+
+      return {
+        ...store,
+        clickAllThreadPoolTracks,
+        profile,
+      };
+    }
+
+    // This runs 2 tests: the first right clicks a global track, the second
+    // right clicks the local track.
+    it.each([
+      { type: 'global', trackIndex: 0 },
+      {
+        type: 'local',
+        pid: 1000,
+        trackIndex: 0,
+      },
+    ])(`from the $type track's context menu`, (rightClickedTrackReference) => {
```

Comment:
oh I didn't know that you could write like `$type`, nice!

Reply:
Note that this isn't JavaScript syntax, this comes with the `test.each` jest API.

Reply:
oh yeah, I assumed so :)